client/src/components/FeedbackObject/index.js

```diff
@@ -7,159 +7,148 @@ import Button from "../Button/Button";
-						return (
-							<div className="previous-feedback-time" key={index}>
-								<input
-									className="previous-feedback-time-input"
-									name={property}
-									value={timeDifference(Date.now(), currentFeedback[property])}
-									onChange={handleEdit}
-									disabled
-								/>
-							</div>
-						);
-					}
-				})}
-			<div className="edit-delete-buttons">
-				<Button
-					content={isEditing ? "Save" : "Edit"}
-					handleClick={() => {
-						setIsEditing(!isEditing);
-						isEditing && saveFeedback(currentFeedback.id);
-					}}
-				/>
-				<div>
-					<Button
-						content="Delete"
-						handleClick={() => {
-							alert("DELETE");
-							handleDelete(currentFeedback.id);
-							updateFeedback();
-						}}
-					/>
-				</div>
-			</div>
-		</div>
-	);
+  return (
+    <div className="prev-feedback-list">
+      {Object.keys(currentFeedback).length &&
+        Object.keys(currentFeedback).map((property, index) => {
+          if (property === "module") {
+            return (
+              <div className="feedback-title-and-module">
+                <div className="feedback-title" key={index}>
+                    <textarea
+                      className="feedback-title-textarea"
+                      name="title"
+                      value={currentFeedback.title}
+                      onChange={handleEdit}
+                      disabled={!isEditing}
+                    />
+                </div>
+                <div className="feedback-module" key={index}>
+                    <h6 className="title-module">Module:</h6>
+                    {!isEditing ? (
+                      <p className="modules-dropdown">
+                        {currentFeedback.module}
+                      </p>
+                    ) : (
+                      <select
+                        name={property}
+                        value={currentFeedback[property]}
+                        onChange={handleEdit}
+                      >
+                        {modules.map((module, index) => (
+                          <option
+                            className="modules-dropdown"
+                            key={index}
+                            value={module.name}
+                          >
+                            {module.name}
+                          </option>
+                        ))}
+                      </select>
+                    )}
+                  </div>
+              </div>
+            );
+          }
+          if (property === "text") {
+            return (
+              <div className="prev-feedback-text" key={index}>
+                <textarea
+                  className="feedback-text-textarea"
+                  name={property}
+                  value={currentFeedback.text}
+                  onChange={handleEdit}
+                  disabled={!isEditing}
+                />
+              </div>
+            );
+          }
+          if (property === "mentor") {
```

Comment:
I think having if (property === "mentor") here is confusing as straight after comes the time. 
Also if you remove the if statment only from here at least you could move the buttons higher (if you want to put given by and time on the left for example)

Reply:
Removing if statement caused bug in filtering and I needed to put them back in.

Reply:
You can at least remove the rendering of the time element outside the if statement if you want to keep it. 
Having if (property === "mentor") and then time straight after it: 
                div className="prev-feedback-time" key={index}
                  {timeDifference(Date.now(), currentFeedback.time)}
Is not right.